susemanager-utils/susemanager-sls/salt/services/docker.sls

```diff
@@ -15,7 +15,7 @@ mgr_install_docker:
 {%- endif %}
 {%- if grains['saltversioninfo'][0] >= 2018 %}
       - python3-salt
-    {%- if salt['pkg.info_available']('python', 'python2') and salt['pkg.info_available']('python2-salt') %}
+    {%- if salt['pkg.info_available']('python', 'python2') and salt['pkg.info_available']('python2-salt') and salt['pkg.info_available']('python-Jinja2', 'python2-Jinja2') %}
```

Comment:
- performance: maybe it's faster to do a series of nested `{% if`s (I'm not sure that jinja short-circuits boolean expressions)
- other possibility: `pkg.info_available` takes several arguments. Are they joined with a OR or a AND? If it's a AND, you could list all packages in one single call

My 2 cents

Reply:
Seems to behave as an OR:
```
juliogonzalez@juliogonzalez:~/Documents/development/obs/suse/Devel:Galaxy:Manager:Head/salt$ sudo salt-call --local pkg.info_available python python2
[sudo] пароль для root: 
local:
    ----------
    python:
        ----------
        arch:
            x86_64
        installed:
            True
        installed size:
            1.4 MiB
        repository:
            openSUSE-Leap-15.1-Update
        source package:
            python-2.7.14-lp151.10.13.1.src
        status:
            up-to-date
        summary:
            Intérprete Python
        vendor:
            openSUSE
        version:
            2.7.14-lp151.10.13.1
juliogonzalez@juliogonzalez:~/Documents/development/obs/suse/Devel:Galaxy:Manager:Head/salt$ sudo salt-call --local pkg.info_available python pythonblah
local:
    ----------
    python:
        ----------
        arch:
            x86_64
        installed:
            True
        installed size:
            1.4 MiB
        repository:
            openSUSE-Leap-15.1-Update
        source package:
            python-2.7.14-lp151.10.13.1.src
        status:
            up-to-date
        summary:
            Intérprete Python
        vendor:
            openSUSE
        version:
            2.7.14-lp151.10.13.1
```

Reply:
Apparently it's a OR, so you can forget second suggestion

Reply:
As for the `if` even if jinja doesn't short-circuits boolean expressions, I think having nested `if`s make things easier to understand. Will change it.

Reply:
After trying, nesting seems to things harder to understand, so let's keep it as it is.

Reply:
I think the interpreter should evaluate the first statement and only if this is true the next statement is executed. So I would suggest to put the Jinja test first.

Reply:
Sorry, didn't notice the comment in time.

I will issue a new PR and merge it right away.